sass/_FullSearchGrid.scss

```diff
@@ -75,6 +75,17 @@ $page-width: 1000px;
       display: none;
     }
   }
+  .coveo-results-footer {
+    &,
+    .coveo-results-footer-left,
+    .coveo-results-footer-right {
+      display: flex;
+      align-items: center;
+    }
+    .coveo-results-footer-left {
+      flex-grow: 1;
+    }
+  }
```

Comment:
Would it have been possible to fix the issue by adjusting existing CSS rules?

I think adding new DOM elements is fine, just curious to know what the alternatives were.

Reply:
It doesn't seem like there's another solution that doesn't involve adding a new element.

It might be possible to transform the `.coveo-results-column` into some sort of flexbox or table and to exchange the position of `.CoveoLogo` & `.CoveoResultsPerPage`, but that would be even more invasive.